bin/commands.sh

```diff
@@ -67,6 +67,11 @@ copysrc() {
   echo "Copying docs-config.js into .docs"
   cp -r ../docs-config.js ./
 
+  if [ -e ../static ]; then
+      echo "Copying static into .docs"
+      cp -r ../static ./
```

Comment:
Given that the bucket is `.docs/public` (https://github.com/cloudflare/cloudflare-docs/blob/572839b8c9621336d012a3c67ee602fa22d194d1/products/workers/wrangler.toml#L15), doesn’t this have to be `cp -r ../static ./public`? But the problem with that is then it would be blown away by Gatsby on develop/build. So doesn’t that mean this needs to be somewhere else? I may be misunderstanding what this is or how it works though.

Reply:
@signalnerve Would love your help investigating/testing this.

Reply:
I think the way this works is that we're turning `.docs` into a Gatsby project folder — once Gatsby starts the build process, it'll manage the `static` folder and merge whatever's in there into `public`.